assets/js/copyable.js

```diff
@@ -0,0 +1,33 @@
+async function copyText(e) {
+    const btn = e.currentTarget;
+    const icon = btn.firstChild;
+    const textNode = btn.previousElementSibling;
+    try {
+        await navigator.clipboard.writeText(textNode.textContent);
+        icon.classList.remove('fa-copy');
+        icon.classList.add('fa-check');
+      } catch (err) {
+        alert(`Error copying text to clipboard: ${err.message}\n\nPlease copy manually instead.`);
+        const selection = window.getSelection();
+        const range = document.createRange();
+        range.selectNodeContents(textNode);
+        selection.removeAllRanges();
+        selection.addRange(range);
+    }
+    setTimeout(() => {
+        icon.classList.remove('fa-check');
+        icon.classList.add('fa-copy');
+      }, 1000);
```

Comment:
Looks like I messed up the indentation on this line.

```suggestion
    }, 1000);
```